torchci/lib/fetchFlakyTests.ts

```diff
@@ -111,7 +111,14 @@ select
             if(TYPEOF(t.rerun) = 'object', 1, Length(t.rerun)),
             if(TYPEOF(t.rerun) = 'object', 2, Length(t.rerun) + 1)
         )
-    ) as numRed
+    ) as numRed,
+    ARBITRARY(
+      if(
+          TYPEOF(t.rerun) = 'object',
+          t.rerun.text,
+          t.rerun[1].text
```

Comment:
Why `rerun[1]` here?
```suggestion
          t.rerun[0].text
```

Reply:
Apparently its 1 indexed